squirrel_attention/src/AttentionFusion.cpp

```diff
@@ -143,7 +143,21 @@ void AttentionFusion::legsCallback(const people_msgs::People& msg)
     observeMutex_.lock();
     reason_ = "leg";
     observeMutex_.unlock();
-  }
+
+    robotino_msgs::LookAtPosition srv;
+    srv.request.target = next_.point;
+    srv.request.why = reason_;
```

Comment:
for the current version `robotino_msgs` ( https://github.com/squirrel-project/squirrel_common/blob/indigo_dev/robotino_msgs/srv/LookAtPosition.srv)  version:

`srv.request.reason = reason_;`

Reply:
Seems some old code slipped through. Will send an update later.